docs/modules/user-guide/partials/proc_adding-kubernetes-component-to-a-devfile.adoc

```diff
@@ -12,116 +12,45 @@ This section describes how to add a `kubernetes` component to a devfile. A compl
-+
-.Overriding container entrypoints
+.Adding `kubernetes` component using the `inlined` property
 ====
 [source,yaml]
 ----
   components:
-    - name: appDeployment
-      kubernetes:
-        reference: app-deployment.yaml
-        entrypoints:
-        - parentName: mysqlServer
-          command: ['sleep']
-          args: ['infinity']
-        - parentSelector:
-          app: prometheus
-          args: ['-f', '/opt/app/prometheus-config.yaml']
+  - name: myk8deploy
+    kubernetes:
+      inlined: |
+        apiVersion: batch/v1
+        kind: Job
+        metadata:
+          name: pi
+        spec:
+          template:
+            spec:
+              containers:
+              - name: job
+                image: myimage
+                command: ["some",  "command"]
+              restartPolicy: Never
 ----
 ====
 +
-The `entrypoints` list contains constraints for picking the containers along with the `command` and `args` parameters to apply to them. In the example above, the constraint is `parentName: mysqlServer`, which will cause the command to be applied to all containers defined in any parent object called `mysqlServer`. The parent object is assumed to be a top level object in the list defined in the referenced file, which is `app-deployment.yaml` in the example above.
-+
-Other types of constraints (and their combinations) are possible:
-+
-`containerName`:: the name of the container
-`parentName`:: the name of the parent object that (indirectly) contains the containers to override
-`parentSelector`:: the set of labels the parent object needs to have
-+
-A combination of these constraints can be used to precisely locate the containers inside the referenced Kubernetes list.
+. `Kubernetes` or `openshift` components allow you to specify the endpoint through the endpoint property.
 
-. Overriding container environment variables
-+
-To provision or override entrypoints in a Kubernetes or OpensShift component, configure it in the following way:
-+
-.Overriding container environment variables
-====
-[source,yaml]
-----
-  components:
-    - name: appDeployment
-      kubernetes:
-        reference: app-deployment.yaml
-        env:
-          - name: ENV_VAR
-            value: value
-----
-====
-+
-This is useful for temporary content or without access to editing the referenced content. The specified environment variables are provisioned into each init container and containers inside all Pods and Deployments.
-
-. Specifying mount-source option
-+
-To specify a project sources directory mount into container(s), use the `mountSources` parameter:
-+
-.Specifying mount-source option
-====
-[source,yaml]
-----
-   components:
-      - name: appDeployment
-        kubernetes:
-          reference: app-deployment.yaml
-          mountSources: true
-----
-====
+. `Kubernetes` or `openshift` components can be associated with `Apply` commands. If no `Apply` commands are associated, they are assumed to be applied at the start up.
```

Comment:
Are these commentaries or instructions? The passive voice and absence of imperatives make this unclear.